src/FakeItEasy/Core/DefaultFakeManagerAccessor.cs

```diff
@@ -1,4 +1,4 @@
-namespace FakeItEasy.Core
+﻿namespace FakeItEasy.Core

```

Comment:
This is probably a change in file encoding. Does your local diff viewer tell you that the left side is one encoding, e.g. UTF-8, and the right side is something else?

It's not huge issue, but it would nice to avoid this change if possible.